apps/api/src/yjs/v2/executor/convertToNotebook.ts

```
@@ -0,0 +1,263 @@
+          source: sqlBlock.source?.toJSON() ?? '',
+          metadata: {
+            language: 'sql',
+            variable: sqlBlock.dataframeName.value,
+            id: cellId
+          },
+          execution_count: null,
+          outputs: [],
+        })
+        break
+      case 'INPUT':
+        const inputBlock = getInputAttributes(currentBlock as Y.XmlElement<InputBlock>, blocks)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)